helmfile/elastisys/opendistro-es/files/configurer/index-templates/authlog.template.json

```diff
@@ -5,7 +5,6 @@
   "template": {
     "settings": {
       "index.refresh_interval": "10s",
-      "opendistro.index_state_management.policy_id": "authlog",
       "opendistro.index_state_management.rollover_alias": "authlog",
```

Comment:
did you notice any issues caused by this name change?

Reply:
This setting `policy_id` is deprecated in favor for adding `index_patterns` under `ism_template` in the policy instead.
That is why it is removed. I did verify that it will apply after an rollover so it does work.